[2024-10-07T13:04:12.977+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-10-07T13:04:12.988+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: sql_dag.check_if_table_exists manual__2024-10-07T13:04:08.984659+00:00 [queued]>
[2024-10-07T13:04:12.993+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: sql_dag.check_if_table_exists manual__2024-10-07T13:04:08.984659+00:00 [queued]>
[2024-10-07T13:04:12.993+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 1
[2024-10-07T13:04:13.000+0000] {taskinstance.py:2888} INFO - Executing <Task(BranchPythonOperator): check_if_table_exists> on 2024-10-07 13:04:08.984659+00:00
[2024-10-07T13:04:13.003+0000] {warnings.py:112} WARNING - /home/airflow/.local/lib/python3.12/site-packages/airflow/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=3756) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-10-07T13:04:13.004+0000] {standard_task_runner.py:72} INFO - Started process 3758 to run task
[2024-10-07T13:04:13.005+0000] {standard_task_runner.py:104} INFO - Running: ['airflow', 'tasks', 'run', 'sql_dag', 'check_if_table_exists', 'manual__2024-10-07T13:04:08.984659+00:00', '--job-id', '185', '--raw', '--subdir', 'DAGS_FOLDER/sql.py', '--cfg-path', '/tmp/tmpqqi6jifn']
[2024-10-07T13:04:13.005+0000] {standard_task_runner.py:105} INFO - Job 185: Subtask check_if_table_exists
[2024-10-07T13:04:13.036+0000] {task_command.py:467} INFO - Running <TaskInstance: sql_dag.check_if_table_exists manual__2024-10-07T13:04:08.984659+00:00 [running]> on host ea4b7cb1f1d7
[2024-10-07T13:04:13.087+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='sql_dag' AIRFLOW_CTX_TASK_ID='check_if_table_exists' AIRFLOW_CTX_EXECUTION_DATE='2024-10-07T13:04:08.984659+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-10-07T13:04:08.984659+00:00'
[2024-10-07T13:04:13.089+0000] {logging_mixin.py:190} INFO - Task instance is in running state
[2024-10-07T13:04:13.089+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: queued
[2024-10-07T13:04:13.089+0000] {logging_mixin.py:190} INFO - Current task name:check_if_table_exists state:running start_date:2024-10-07 13:04:12.989121+00:00
[2024-10-07T13:04:13.090+0000] {logging_mixin.py:190} INFO - Dag name:sql_dag and current dag run status:running
[2024-10-07T13:04:13.090+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-10-07T13:04:13.090+0000] {baseoperator.py:405} WARNING - BranchPythonOperator.execute cannot be called outside TaskInstance!
[2024-10-07T13:04:13.099+0000] {base.py:84} INFO - Retrieving connection 'postgres_default'
[2024-10-07T13:04:13.106+0000] {sql.py:509} INFO - Running statement: 
                    SELECT COUNT(*)
                    FROM information_schema.tables
                    WHERE table_name = 'table_name'
                    , parameters: None
[2024-10-07T13:04:13.111+0000] {sql.py:518} INFO - Rows affected: 1
[2024-10-07T13:04:13.112+0000] {logging_mixin.py:190} INFO - (1,)
[2024-10-07T13:04:13.113+0000] {python.py:240} INFO - Done. Returned value was: dummy_operator
[2024-10-07T13:04:13.114+0000] {branch.py:38} INFO - Branch into dummy_operator
[2024-10-07T13:04:13.115+0000] {skipmixin.py:230} INFO - Following branch ('dummy_operator',)
[2024-10-07T13:04:13.127+0000] {skipmixin.py:278} INFO - Skipping tasks [('create_table', -1)]
[2024-10-07T13:04:13.164+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-10-07T13:04:13.165+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=sql_dag, task_id=check_if_table_exists, run_id=manual__2024-10-07T13:04:08.984659+00:00, execution_date=20241007T130408, start_date=20241007T130412, end_date=20241007T130413
[2024-10-07T13:04:13.172+0000] {logging_mixin.py:190} INFO - Task instance in success state
[2024-10-07T13:04:13.172+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: running
[2024-10-07T13:04:13.173+0000] {logging_mixin.py:190} INFO - Dag name:sql_dag queued_at:2024-10-07 13:04:09.010155+00:00
[2024-10-07T13:04:13.173+0000] {logging_mixin.py:190} INFO - Task hostname:ea4b7cb1f1d7 operator:BranchPythonOperator
[2024-10-07T13:04:13.184+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-10-07T13:04:13.206+0000] {taskinstance.py:3900} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-10-07T13:04:13.207+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
